trimmer_decorator.rb

```diff
@@ -0,0 +1,6 @@
+require '.\decorator'
```

Comment:
- [ ] Let me explain what the other reviewer meant here.  `require './nameable' and require_relative './nameable'` are not the same.

- `require './nameable'` searches for the file `nameable.rb` in the current directory. It uses an absolute path, starting from the current working directory.

- On the other hand, `require_relative './nameable'` searches for the file `nameable.rb` relative to the current file's location. It uses a relative path, starting from the directory where the current file is located.

- When you are working on a project with multiple files, it's generally better to use `require_relative` because it provides a more reliable and portable way of loading files within the project's directory structure. Using `require_relative` helps avoid issues when the current working directory changes or when the code needs to be run from a different location.

- However, if you are working with a standalone script or the file you are requiring is located outside of the current project directory, you can use require './nameable' to load the file using an absolute path. But that's not the case here 👍

Reply:
Hi! @Shedrack-Sunday, thanks for telling me what is the difference but the thing is that I have completed everything according to the instruction and I think this is not a requirement in the project. But anyways, I have made the changes. 😉